pkg/cryptoauthlib/doc.txt

```diff
@@ -0,0 +1,54 @@
+/**
+ * @defgroup pkg_cryptoauthlib Microchip CryptoAuthentication Library
+ * @ingroup  pkg drivers
+ * @brief    Provides the library for Microchip CryptoAuth devices
+ * @see      https://github.com/MicrochipTech/cryptoauthlib
+ *
+ * # Introduction
+ *
+ * This package provides support for the official library for Microchip CryptoAuth devices.
+ *
+ *
+ * ## Warning
+ *
+ * Some functions can only be used, when the data, config and otp zones of the
+ * device are locked. Locking is permanent and cannot be undone. Be careful if
+ * you're not sure you've configured everything correctly.
+ * For more information we refer to the data sheet of the device.
+ *
+ *
+ * ## Usage
+ *
+ * Add
+ * USEPKG += cryptoauthlib
+ * to your Makefile.
+ *
+ *
+ * ## Implementation status
+ *
+ * This implementation was partly tested with the ATECC508A. We haven't tested the
+ * functions that require locking the device, yet. There's a wrapper in the
+ * cryptoauthlib/contrib folder, which implements most of the HAL functions.
+ * Currently the functions hal_i2c_discover_devices and hal_i2c_discover_buses
+ * are unimplemented, as well as hal_i2c_post_init.
+ *
+ * ### Wake function
+ *
+ * The wake function only works if the baudrate is 133 kHz or less.
```

Comment:
When a 0x00 byte is sent on an i2c interface that runs with with 133kHz or less.